docs/reference/modules/asset.md

```diff
@@ -0,0 +1,205 @@
+---
+extends: '@apostrophecms/module'
+---
+
+# `@apostrophecms/asset`
+
+**Alias:** `apos.asset`
+
+<AposRefExtends :module="$frontmatter.extends" />
+
+The `asset` module serves to organize, process, and output all project script and styling assets during the build process. In addition, it provides access to modify the project webpack configuration and exposes two CLI tasks for project building and webpack cache clearing. Options are passed through the creation of a `modules/@apostrophecms/asset/index.js` file.
+
+## Options
+
+|  Property | Type | Description |
+|---|---|---|
+| [`refreshOnRestart`](#refreshonrestart) | Boolean | If set to `true`, the browser will refresh on Apostrophe app restart. |
+| [`watch`](#watch) | Boolean | If set to `false`, none of the UI assets will be watched for triggering a restart. |
+| `watchDebounceMs` | Integer | Time in milliseconds to wait before re-triggering a restart on asset change. |
+
+### `refreshOnRestart`
+
+If `refreshOnRestart` is truthy and `process.env.NODE_ENV` is not set to `production`, then restart of the Apostrophe app will trigger a browser refresh.
+
+### `watch`
+
+If `watch` is falsy, then changes to any of the assets in the `modules` and `node_modules` will not trigger a restart. Note that this option is disabled and has no impact if `process.env.NODE_ENV` is set to `production`. 
```

Comment:
It's not a restart, it's a webpack rebuild of the asset bundles without restarting the entire Apostrophe application. Also always document what the default behavior is.

Reply:
Changed.